fah-data-cleanup.pl

```diff
@@ -5,44 +5,55 @@
 use Cwd;
 use Getopt::Long qw(HelpMessage :config pass_through);
 
-my $proj     = $ARGV[0] or die HelpMessage();
-my $maxrun   = $ARGV[1] or die HelpMessage();
-my $maxclone = $ARGV[2] or die HelpMessage();
-
-my $currentrun = 0;
-my $homedir    = getcwd();
-
-while ($currentrun < $maxrun) {
-    my $currentclone = 0;
-    while ($currentclone < $maxclone) {
-        my $workdir = "$homedir/PROJ$proj/RUN$currentrun/CLONE$currentclone/";
-        chdir $workdir;
-        my $test = getcwd();
-        print STDOUT "Working on directory $test ...\n";
-        `rm *# *.xvg *.pdb *.out *.nat *.nat6 temp* 2> /dev/null`;
-        `mv frame0.tpr temp`;
-        `rm *.tpr 2> /dev/null`;
-        `mv temp frame0.tpr`;
-        `mv ener.edr temp`;
-        `rm *.edr 2> /dev/null`;
-        `mv temp ener.edr`;
-        $currentclone++;
+my $projpath = $ARGV[0] or die HelpMessage();
+
+if (-d $projpath) {
+    my @runs = &pattern_walk("RUN", $projpath);
+    foreach my $run (@runs) {
+        my @clones = &pattern_walk("CLONE", $run);
+        foreach my $clone (@clones) {
+            print STDOUT "Working on directory $clone ...\n";
+            chdir $clone;
+            `rm *# *.xvg *.pdb *.out *.nat *.nat6 *.txt temp* 2> /dev/null`;
+            `mv frame0.tpr temp`;
+            `rm *.tpr 2> /dev/null`;
+            `mv temp frame0.tpr`;
+            `mv ener.edr temp`;
+            `rm *.edr 2> /dev/null`;
+            `mv temp ener.edr`;
+        }
     }
-    $currentrun++;
+} else {
+    print STDOUT "$projpath: No such file or directory\n";
+    exit 1;
 }
-
 print STDOUT "Done!\n";
 
+# Arguments: pattern to search for, absolute path of directory
+sub pattern_walk {
+    my ($pattern, $path, @dirs, $num_dirs);
+    $pattern  = $_[0];
+    $path     = $_[1];
+    @dirs     = `ls $path | grep $pattern`;
+    $num_dirs = scalar(@dirs);
+    for (my $x = 0 ; $x < $num_dirs ; $x++) {
+        my $dir = $dirs[$x];
+        chomp $dir;
+        $dirs[$x] = "$path/$dir";
+    }
+    return @dirs;
+}
+
 =head1 NAME
 
 fah-data-clean-up.pl - Remove unwanted files from a F@H dataset
 
 =head1 SYNOPSIS
 
-./fah-data-clean-up.pl <project_dir> <#_of_runs> <#_of_clones>
+./fah-data-clean-up.pl <Absolute path to PROJ# directory>
```

Comment:
Can I use relative path?

Also please use something like `<path_to_project_dir>` for style consistency in POD.